README.md

```diff
@@ -8,17 +8,24 @@ please see the
 [mongodb/mongo-bi-connector-odbc-driver](https://github.com/mongodb/mongo-bi-connector-odbc-driver)
 repository.
 
+## Verify Integrity of MongoDB ODBC Driver packages
+You can download the [MongoDB ODBC Driver](https://www.mongodb.com/try/download/odbc-driver) from MongoDB Download Center.   
+Select the platform and version you want to verify on that page.   
+Click `Copy link` and use the URL while following the instructions to validate MongoDB packages described [here](https://www.mongodb.com/docs/manual/tutorial/verify-mongodb-packages/).
 
-## Set up the ODBC driver on Windows
+> :warning:  
+> For this [step](!https://www.mongodb.com/docs/manual/tutorial/verify-mongodb-packages/#download-then-import-the-key-file), download and import the MongoDB ODBC Driver public key using this url : `https://pgp.mongodb.com/atlas-sql-odbc.asc`
+
+## Manually set up the ODBC driver on Windows for local development
```

Comment:
Just for consistency we may want to choose a style for titles, either Title Case or regular sentence.